dotcom-rendering/src/components/SecureReCAPTCHASignup.tsx

```diff
@@ -219,7 +219,9 @@ export const SecureReCAPTCHASignup = ({
 	successDescription,
 }: Props) => {
 	const recaptchaRef = useRef<ReactGoogleRecaptcha>(null);
-
+	const [isCaptchaSiteKey, setIsCaptchaSiteKey] = useState<
+		string | undefined
+	>();
```

Comment:
FYI the `undefined` can be inferred automatically.

```suggestion
	const [isCaptchaSiteKey, setIsCaptchaSiteKey] = useState<string>();
```

Also, what is the benefit of the `is` prefix?

Reply:
Thanks Max, type updated and `is` prefix removed as its not needed.